cmd/mayactl/app/command/commands.go

```diff
@@ -16,17 +16,32 @@ package command
 
 import (
 	"flag"
+	"fmt"
+	"os"
 
 	"github.com/openebs/maya/cmd/mayactl/app/command/snapshot"
+	"github.com/openebs/maya/pkg/client/mapiserver"
 	"github.com/spf13/cobra"
 )
 
-// NewCommand creates the `maya` command and its nested children.
+// NewMayaCommand creates the `maya` command and its nested children.
 func NewMayaCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "mayactl",
-		Short: "Maya means 'Magic' a tool for storage orchestration",
+		Short: "Maya means 'Magic'a tool for storage orchestration",
 		Long:  `Maya means 'Magic' a tool for storage orchestration`,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if mapiserver.MAPIAddr == "" {
```

Comment:
can you make it len(mapiserver.MAPIAddr) == 0  , doing this way makes code clear.

Reply:
sure